clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2728,6 +2728,23 @@ static void emitKernelNameType(QualType T, ASTContext &Ctx, raw_ostream &OS,
   emitWithoutAnonNamespaces(OS, T.getCanonicalType().getAsString(TypePolicy));
 }
 
+int SYCLIntegrationHeader::getCppVersion() {
+  LangOptions LangOpts = S.getASTContext().getLangOpts();
+  if (LangOpts.CPlusPlus) {
```

Comment:
I'd likely replace this condition with an assert.

Reply:
Updated!